test/xml_parser.js

```diff
@@ -56,6 +56,12 @@ module.exports.recoverable_parse = function(assert) {
 };
 
 module.exports.baseurl_xml = function(assert) {
+    if (/^win/.test(process.platform)) {
```

Comment:
Is there a bug report to track this? If so it would be nice to see its number cross referenced, perhaps both here in the discussion and in the source code comment.